### ISSUE TITLE
pub get will not get mysql1 package

### ISSUE THREAD

**Issue by psionprime** [dart-lang/pub-dev#3651]
Dart v2.8.2, package mysql1 0.17.1

Placing a dependency in the yaml file as I have for others seems to work with pub get, but mysql1 is not:
- in the import package list in VSCode
- in package cache
- in .package
- in pub deps listing

Using pub cache add mysql1 indicates it has downloaded but again, all the above. Where is the logging of the actions to try and trace what is going on here ? 

I have pub cache repaired (which killed stagehand and dart tools btw) and it is the same. I can add dependencies like crypto and they work fine. I get mysql1 and it appears to work but the files are no where on my system so there is some sort of silent fail going on I'd like to track down.

**Comment by isoos** [comment]
@psionprime: Could you please include the logs of `pub get -v`?

**Comment by psionprime** [comment]
I powered down, came back today and now it is listed in 'pub deps'. I spent hours trying to properly install this with almost no feedback during the process. I'm going to close this issue and see about using the library. If this comes up again I will use 'pub get -v'. Perhaps some more obvious logging or messaging during the get to indicate success or fail will be helpful. Thank you for your efforts.
